wiki/Ranking_criteria/Metadata/en.md

```diff
@@ -0,0 +1,266 @@
+# Metadata
+
+## General
+
+### Rules
+
+- **[Primary metadata sources](/wiki/Beatmap/Primary_metadata_source) must be used as reference.** You may modify metadata from primary sources only in ways allowed or required by the Ranking Criteria. If no primary sources are available, use what is most recognisable.
+- **All difficulties in a beatmap set must have identical `Title`, `Artist`, `Tag`, `Source`, and `BeatmapSetID` fields.**
+- **`Artist` or `Title` fields over 81 characters must be shortened.** <!-- Note: this rule can be dropped with no replacement if lazer allows for more flexible metadata uploads -->
+  - Start by dropping additional markers.
+  - If not sufficient, truncate the field and indicate this by using `...` in a sensible place.
+  - Omitted information must be added to tags.
+- **The `Tags` field must be shortened if it is longer than 1000 characters.** <!-- This needs to be adjusted if this limit changes or is removed. -->
+  - Start by dropping optional tags.
+  - Keep required tags such as usernames as much as possible.
+  - For the rest, prioritize the tags that are most relevant for search. 
+
+### Guidelines
+
+- **Logos should not be used as reference for capitalisation.** As logos are often stylised, apply regular title case[^title-case] unless other textual metadata supports it.
+- **When a remix or cover appears to have a typo in the song name compared to the original track, use the original spelling instead, unless the typo is an intentional stylisation.** If it is unclear, a [discussion](/wiki/Beatmap_discussion#discussions) should be held to come to a consensus on which to use. Below are examples with a remix of a song titled `triangles`:
+  - `triangls` should be `triangles`
+  - `triANGELS (angelic remix)` should be left as-is
+  - `3angle5` should be left as-is
+  - `Triangles` can be either `Triangles` or `triangles`
+- **When multiple metadata options are available:**
+  - Aim to match Ranked or Loved beatmaps. Follow what is most recent and common, then verify that metadata is correct and fix as needed.
```

Comment:
```suggestion
  - Aim to match Ranked beatmaps. Follow what is most recent and common, then verify that metadata is correct and fix as needed.
```

Loved isn't as able to enforce every requested change as it used to be, so may be best to change this accordingly.